tools/code-owners-parser/CodeOwnersParser/CodeOwnerEntry.cs

```diff
@@ -130,5 +133,32 @@ private string ParsePath(string line)
             // remove the path from the string.
             return line.Substring(ownerStartPosition);
         }
+
+        public void FilterOutTeamAlias()
+        {
+            client.BaseAddress = new Uri("https://api.github.com/");
+            client.DefaultRequestHeaders.UserAgent.Add(new System.Net.Http.Headers.ProductInfoHeaderValue("CodeOwnerRetriever", "1.0"));
+            client.DefaultRequestHeaders.Accept.Add(new System.Net.Http.Headers.MediaTypeWithQualityHeaderValue("application/json"));
+            Owners.RemoveAll(r => IsTeamAlias(r));
+        }
+
+        private static bool IsTeamAlias(string alias)
+        {
+            var userUriStub = $"users/{alias}";
+            try
+            {
+                var response = client.GetAsync(userUriStub);
+                if (response.Result.IsSuccessStatusCode)
+                {
+                    return false;
+                }
+                return true;
+            }
+            catch
+            {
+                Console.WriteLine($"Http call {userUriStub} to failed.");
```

Comment:
Let's not write to the console.

Reply:
Do we want to throw http native error message?

Reply:
This is error flow. Try to print custom error message. If removed, then we do not need to have try block. Do you mean this?

Reply:
Yes just let the exception flow for now. We might need to consider retries later if we find this isn't reliable enough.